pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -30,6 +31,7 @@ import (
 
 const (
 	operand = "knative-serving"
+	path    = "deploy/resources/webhook/webhook.yaml"
```

Comment:
Name this const "webhookPath" for clarity?

Reply:
Done.
Thanks